index.js

```diff
@@ -10,11 +10,9 @@ const obj = () => Object.create(null);
 
 // Prevent caching of this module so module.parent is always accurate
 delete require.cache[__filename];
-const parentDir = path.dirname(module.parent.filename);
```

Comment:
🤔 i was removing it since it is unused and it's parameter is finally falling back to `.`. It is - of course - no problem to create that const again. But it would require some kind of block statement to avoid having a null-value being evaluated. In addition to that, i'd like to avoid having `parentDir` containing two different values (either the parent directory or a reference to the current working directory).

I could imagine to solve it like this:
```
const parentDir = module.parent.filename === null ? null : path.dirname(module.parent.filename);
...
pkgUp.sync(parentDir || '.');
```

What would you say (I'd like to avoid discussing this after commits)?